device_tags_python/Dockerfile.template

```diff
@@ -0,0 +1,28 @@
+# base-image for python on any machine using a template variable,
+# see more about dockerfile templates here: https://www.balena.io/docs/learn/develop/dockerfile/
+FROM balenalib/%%BALENA_ARCH%%-debian-python:3-bookworm
```

Comment:
Should we really be adding new examples with base images? is there any reason not to use pure python mantained docker?

Reply:
I would keep it in sync w/ the Dockerfile for the node app for now and we can switch them both in a separate PR.